installing/installing_ocp/install-secured-cluster-ocp.adoc

```diff
@@ -69,8 +67,11 @@ include::modules/change-config-options-after-deployment.adoc[leveloffset=+2]
 
 To install {product-title-short} on secured clusters by using the CLI, perform the following steps:
```

Comment:
I know this is existing text. But, do these steps actually install RHACS on secured clusters, or are they things you need to do before the actual install?

Reply:
Good point, I guess technically installing the CLI and generating the files are prerequisites for the last step. I'll revise.